src/registry_schemas/example_data/schema_data.py

```diff
@@ -2701,6 +2701,19 @@
     'freeze': True
 }
 
+AGM_LOCATION_CHANGE = {
+    'year': '2023',
+    'newAgmLocation': {
+        'streetAddress': 'mailing_address - address line one',
+        'streetAddressAdditional': '',
+        'addressCity': 'mailing_address city',
+        'addressCountry': 'CA',
+        'postalCode': 'H0H0H0',
+        'addressRegion': 'ON',
```

Comment:
Is this allowed to be any country & province/state?
I ask as you example is not in BC

Reply:
The `addressRegion` can't be BC.  Vysakh will be adding validation in the legal api for this.